cmd/samples/complex/sender/main.go

```diff
@@ -77,11 +76,8 @@ func (d *Demo) Send() error {
 }
 
 func (d *Demo) context() cloudevents.EventContext {
-	now := time.Now()
```

Comment:
I like the addition of the default ID and Time. But for the purposes of this sample, should there be examples of both using the default (as best practice) and also the longer form for those that might need to assign a specific time?

Reply:
The simple http sender does this, this is the complex example